pkgs/tools/misc/xdiskusage/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl, fltk }:
+
+stdenv.mkDerivation rec {
+  pname = "xdiskusage";
+  version = "1.60";
+
+  src = fetchurl {
+    urls = [
+      "https://xdiskusage.sourceforge.net/${pname}-${version}.tgz"
```

Do not parameterize `pname`:

```suggestion
      "https://xdiskusage.sourceforge.net/xdiskusag -${version}.tgz"
```